vic/drivers/shared_all/src/calc_root_fraction.c

```diff
@@ -124,6 +134,7 @@ calc_root_fractions(veg_con_struct  *veg_con,
                 }
             }
             else if (Zsum + Zstep > Lsum) {
+                zone++;   // should this be here?
```

Comment:
No idea - shouldn't that be up to the person changing the code? At the very least the comment should be resolved (and removed) before the PR is accepted.